jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -131,10 +132,9 @@ pipeline {
         always {
             script {
                 final Map msg = readJSON text: CI_MESSAGE
-                String mailTo = 'cloud-services-delorean@redhat.com'
                 mail(
                         subject: "New build of ${msg.info.name} created in Brew",
-                        to: mailTo,
+                        to: "${mailTo}",
```

Comment:
Is this change needed?

Reply:
No. Thought it had failed on that earlier. But tested it again and it works without it. Successful build here: https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Delorean%202.0/job/delorean-2-ews/job/3scale/job/lfdiscovery/51/console